sql/CMakeLists.txt

```diff
@@ -89,13 +27,33 @@ set(MOD_FILES
     updates/2.2.0--2.2.1.sql
     updates/2.2.1--2.3.0.sql)
 
+# Files for downgrade scripts.
+set(REV_FILES)
+
 set(MODULE_PATHNAME "$libdir/timescaledb-${PROJECT_VERSION_MOD}")
 set(LOADER_PATHNAME "$libdir/timescaledb")
 
 set(TS_MODULE_PATHNAME
     ${MODULE_PATHNAME}
     PARENT_SCOPE)
 
+if(NOT GENERATE_DOWNGRADE_SCRIPT)
+  message(
+    STATUS "Not generating downgrade script: downgrade generation disabled.")
+elseif(NOT GIT_FOUND)
+  message(STATUS "Not generating downgrade script: Git not available.")
+else()
+  generate_downgrade_script(
+    SOURCE_VERSION
+    ${PROJECT_VERSION_MOD}
+    TARGET_VERSION
+    ${UPDATE_FROM_VERSION}
+    INPUT_DIRECTORY
+    ${CMAKE_CURRENT_SOURCE_DIR}/updates
+    FILES
+    reverse-dev.sql)
```

Comment:
This will not work when the release is tagged and the file renamed. Question is if we should generate a downgrade script for all scripts in `REV_FILES` or just for the latest one.

Reply:
Added code that also generate downgrade files for any files in `REV_FILES` and fetch the version from the file name.